server/push/fcm/payload.go

```diff
@@ -597,10 +196,11 @@ func PrepareV1Notifications(rcpt *push.Receipt, config *configType) []*fcmv1.Mes
 		// We don't know the platform of the receiver, must provide payload for all platforms.
 		msg.Android = androidNotificationConfig(rcpt.Payload.What, topic, userData, config)
 		msg.Apns = apnsNotificationConfig(rcpt.Payload.What, topic, userData, 0, config)
+		// TODO: add webpush payload.
 		messages = append(messages, &msg)
 	}
 
-	return messages
+	return messages, uids
```

Comment:
Is `len(messages) == len(uids)` an invariant here?
If it is, channel messages (line 200) breaks it.

Reply:
It should although the difference does not affect anything. Fixed anyway.